configs/fourdag/campus_config/eval_keypoints3d_fourdag_19.py

```diff
@@ -0,0 +1,81 @@
+type = 'BottomUpAssociationEvaluation'
+
+__data_root__ = '../data/CampusSeq1'
```

Comment:
Could we use the same path definition like [mvpose](https://github.com/openxrlab/xrmocap/blob/main/configs/mvpose/campus_config/eval_keypoints3d.py#L3) and [MvP](https://github.com/openxrlab/xrmocap/blob/main/configs/mvp/campus_config/mvp_campus.py#L55)?
You may have put the files somewhere else, but we can make a soft link and access the data under `./xrmocap_data`, which users can find at [docs](https://github.com/openxrlab/xrmocap/tree/main/configs/mvpose).

Reply:
It's `ln -s ../data/CampusSeq1 ./xrmocap/CampusSeq1`.